app/src/main/java/com/fantastic/bookxchange/models/Book.java

```diff
@@ -0,0 +1,48 @@
+package com.fantastic.bookxchange.models;
+
+import org.json.JSONObject;
+import org.parceler.Parcel;
+
+/**
+ * Created by m3libea on 10/11/17.
+ */
+
+@Parcel
+public class Book {
```

Comment:
do you think we need short description field?

Reply:
mmmm yes, mainly for the Book Detail Activity. I'll change that :)